src/main/java/callhub/connect/data_access/LocalDataAccess.java

```diff
@@ -0,0 +1,31 @@
+package callhub.connect.data_access;
+import callhub.connect.entities.exceptions.FileLimitExceededException;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import org.bson.types.Binary;
+
+public class LocalDataAccess {
+    public static byte[] convertPDFToByteArray(String filePath) throws IOException {
+        File pdfFile = new File(filePath);
+        // 1 mb = 1024 * 1024
+
+        try (FileInputStream fileInputStream = new FileInputStream(pdfFile)) {
+            if (pdfFile.length() > 16 * 1024 * 1024) {
```

Comment:
Can we make 16 * 1024 * 1024 a constant?
Eg. 
MB_TO_BITS = 1024 * 1024
fileSizeLimit = 16 * MB_TO_BITS